package.json

```diff
@@ -11,14 +11,15 @@
     "build:es": "rm -rf es && cross-env BABEL_ENV=es babel src -d es --extensions .ts,.tsx --ignore src/**/*.test.tsx",
     "build:ts": "tsc --rootDir src -d --declarationDir lib && tsc --rootDir src -d --declarationDir es",
     "build:icon": "rm -rf src/General/Icon/components && babel-node --extensions .ts,.tsx src/Utils/Icon/generateIcons.ts && npm run format:icon",
-    "format": "prettier --write src/**/*.{ts,tsx} stories/**/*.{ts,tsx}",
     "format:icon": "prettier --write src/General/Icon/components/*.tsx",
-    "prepublishOnly": "npm run build",
+    "prepublishOnly": "npm run lint && npm run test && npm run build && npm whoami && npm version patch -m 'Bump to %s'",
+    "postpublish": "git push --tags",
```

Comment:
how does this work? what tag it pushes?

Reply:
`npm version` will create a version commit and tag automatically, so this script is to push 
 the version tag created by `npm version` to the origin.

Reply:
thank you @rosslo 🥇